TradeItIosTicketSDK2/TradeItLauncher.swift

```diff
@@ -19,19 +19,31 @@ protocol OAuthCompletionListener {
     override internal init() {}
 
     public func handleOAuthCallback(
-        onViewController viewController: UIViewController,
+        onViewController safariViewController: UIViewController,
         oAuthCallbackUrl: URL,
         onSuccessfulLink: ((TradeItLinkedBroker) -> Void)? = nil
     ) {
         print("=====> handleOAuthCallback: \(oAuthCallbackUrl.absoluteString)")
 
         let oAuthCallbackUrlParser = TradeItOAuthCallbackUrlParser(oAuthCallbackUrl: oAuthCallbackUrl)
 
-        self.oAuthCompletionUIFlow.presentOAuthCompletionFlow(
-            fromViewController: viewController,
-            oAuthCallbackUrlParser: oAuthCallbackUrlParser,
-            onSuccessfulLink: onSuccessfulLink
-        )
+        var parentViewController = safariViewController.presentingViewController
```

Comment:
Add a comment about why you need to do this (relinking) as it is very confusing.